README.md

```diff
@@ -13,6 +13,37 @@ application.
 
 <p align="center">Learn all about Porter at <a href="https://porter.sh">porter.sh</a></p>
 
+# <a name="mixins"></a>Porter Mixins
+
+Mixins provide out-of-the-box support for interacting with different tools and services from inside a bundle. You can always create a mixin, or use the exec mixin and a Custom Dockerfile if a custom mixin doesn't exist yet.
+
+[Porter Mixins](https://porter.sh/mixins/) are available for below platform's:
+
+| Platform                                                                                                                                                                                                                        | Supported?  |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :---------: |
+| <img src="docs\static\images\mixins\docker_icon.png" width="20" height="20" vertical-align="middle" /> [Docker](https://porter.sh/mixins/docker/)                                            |     ✔️      |
+| &nbsp;&nbsp;&nbsp; <img src="docs\static\images\mixins\docker-compose.png" width="20" height="20" vertical-align="middle" /> [Docker-Compose](https://porter.sh/mixins/docker-compose/)            |     ✔️      |
```

Comment:
Looks like you forgot the leading whitespace on the first row in the table (for Docker and hashicorp below). Can you either remove the leading whitespace on the other rows, or add the same whitespace to the first row of each table?

Reply:
Hello @carolynvs, I am not getting _leading whitespaces_, can you be more clear!

Reply:
On line 24, in the first table cell you have this

`<img src="docs\static\images\mixins\docker_icon.png" ...`

On line 25 (and the remaining cells in the table), you have the HTML whitespace character `&nbsp;` (stands for non-breaking space) in front of the image, which shifts the image over to the right when the page is displayed.

`&nbsp;&nbsp;&nbsp; <img src="docs\static\images\mixins\docker-compose.png"`

If those whitespace characters `&nbsp;` were unintentional, would you please remove them from both tables?

Reply:
@carolynvs I have pushed the changes do take a look at it!